pkg/pillar/types/global.go

```diff
@@ -295,6 +295,10 @@ const (
 	SyslogLogLevel GlobalSettingKey = "debug.syslog.loglevel"
 	// KernelLogLevel global setting key
 	KernelLogLevel GlobalSettingKey = "debug.kernel.loglevel"
+	// SyslogRemoteLogLevel global setting key
```

Comment:
I don't know why we have this tradition of adding a useless comment to all the properties; it would make sense to stop and add something more meaningful (I'm trying to do it myself).

Reply:
It would also be beneficial to add documentation for the new props in the same commit or in one that's close.
https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md

Reply:
> I don't know why we have this tradition of adding a useless comment to all the properties; it would make sense to stop and add something more meaningful (I'm trying to do it myself).

It is a lint requirement to document all exported names. But we can put something more useful after the // SyslogRemoteLogLevel required prefix.

Reply:
> It would also be beneficial to add documentation for the new props in the same commit or in one that's close.

I'm doing it in my main PR https://github.com/lf-edge/eve/pull/4413

Reply:
I changed the comments to more meaningful descriptions